README.md

```diff
@@ -123,9 +123,12 @@ val scheduler =
 // System.out but to connect with for example sbt server you use sockets: https://github.com/sbt/ipcsocket
 val io = new InputOutput(System.in, System.out)
 
+
 // Construct logger that appends to file, don't log to stdout since that is reserved for
 // JSON-RPC communication.
-val logger = Logger("my-logger").withHandler(writer = scribe.writer.FileWriter.simple())
+val logPath = java.nio.file.Paths.get("my-logger.log").toAbsolutePath
+val fileWriter = scribe.writer.FileWriter().path(timestamp => logPath).autoFlush
+val logger = Logger("my-logger").orphan().withHandler(writer = fileWriter)
```

Comment:
FYI @zainab-ali scribe 2.6.0 changes the `FileWriter` API, here's how to use the new API (it took me a while to figure out).